aiida_registry/test_install.py

```diff
@@ -177,8 +215,19 @@ def test_install_all(container_image):
                 print("    >> SKIPPING: No pip_url key provided")
             continue
 
-        test_results.append(test_install_one_docker(container_image, plugin))
-
-    print(f"Dumping {PLUGINS_TEST_RESULTS}")
-    with open(PLUGINS_TEST_RESULTS, "w", encoding="utf8") as handle:
-        json.dump(test_results, handle, indent=2)
+        results = test_install_one_docker(container_image, plugin)
+        process_metadata = results["process_metadata"]
+        for ep_group in ENTRY_POINT_GROUPS:
+            try:
+                if process_metadata[ep_group]:
+                    for key, _ in data[_k]["entry_points"][ep_group].items():
+                        data[_k]["entry_points"][ep_group][key] = process_metadata[
```

Comment:
looks to me like `_k` is being used and should no longer start with an underscore (perhaps `plugin_name` or something)